worlds/stardew_valley/docs/setup_fr.md

```diff
@@ -0,0 +1,86 @@
+
+## Logiciels requis
+
+- Stardew Valley sur PC (Recommandée: [Version Steam](https://store.steampowered.com/app/413150/Stardew_Valley/))
+- SMAPI ([Chargeur de mods pour Stardew Valley](https://smapi.io/))
+- [Version 5.x.x du mod StardewArchipelago](https://github.com/agilbert1412/StardewArchipelago/releases)
+    - Il est important d'utiliser une version 5.x.x du mod pour jouer à des "seeds" qui y ont été générées. Les versions les plus récentes ne peuvent être utilisées qu'avec les versions les plus récentes du générateur de monde, qui ne sont pas encore supportées par archiepalgo.gg.
+
+## Logiciels facultatifs
+- Archipelago via la [page des versions Archipelago](https://github.com/ArchipelagoMW/Archipelago/releases)
+    - (Uniquement pour le textclient)
+- Autres mods de Stardew Valley depuis [Nexus Mods](https://www.nexusmods.com/stardewvalley)
+    - Il y a des [mods supportés](https://github.com/agilbert1412/StardewArchipelago/blob/5.x.x/Documentation/Supported%20Mods.md) que vous pouvez ajouter à votre dossier YAML pour les inclure avec la randomisation de Archipelago.
+
+    - Il n'est cependant **pas** recommandé d'ajouter des mods non supportés, même s'il est possible de le faire. Les intéractions entre les mods peuvent être imprévisibles, et aucune assistance ne sera proposée si des problèmes surviennent.
+    - Plus vous avez de mods non supportés, et plus ils sont lourds, plus vous avez de chance de causer des problèmes.
+
+## Configurer votre dossier YAML
+
+### Qu'est-ce qu'un dossier YAML et pourquoi il m'en faut un ?
+
+Voir le guide pour configurer un YAML basique: [Guide basique de configuration d'un MultiWrold](/tutorial/Archipelago/setup/en)
+
+### Où puis-je obtenir un dossier YAML ?
+
+Vous pouvez customiser vos paramètres en visitant la [page de paramètres de joueur de Stardew Valley](/games/Stardew%20Valley/player-settings)
+
+## Rejoindre une partie Multimondes
+
+### Installation du mod
+
+- Installez [SMAPI](https://smapi.io/) en suivant les instructions sur leur site internet.
+- Téléchargez et extrayez le mod [StardewArchipelago](https://github.com/agilbert1412/StardewArchipelago/releases) dans votre dossier "Mods" de Stardew Valley.
+- *FACULTATIF*: Si vous voulez lancer votre jeu depuis Steam, ajoutez ce qui suit dans vos options de lancement de Stardew Valley:
+    - "[PATH TO STARDEW VALLEY]\Stardew Valley\StardewModdingAPI.exe" %command%
+- Sinon, lancez simplement "StardewModdingAPI.exe" directement dans votre dossier d'installation.
+- Stardew Valley devrait se lancer seul à côté d'une console qui vous permet de lire les informations de mods et d'intéragir avec certains d'entre eux.
+
+### Se connecter au MultiServer
+
+Lancez Stardew Valley avec SMAPI. Une fois que vous êtes sur l'écran d'accueil de Stardew Valley, créez une nouvelle ferme.
+
+Sur la page de création d'un nouveau personnage, vous verrez trois nouveaux champs, utilisés pour lier votre nouveau personnage au multiworld d'Archipelago.
```

Comment:
"champs, utilisés pour lier" -> "champs qui vont vous permettre de lier"
Rewording

Reply:
"au multiworld" -> "à un multiworld"
Since it is not a unique entity.